android/java/org/chromium/chrome/browser/crypto_wallet/activities/AccountDetailActivity.java

```diff
@@ -217,7 +225,12 @@ public void onAssetClick(BlockchainToken asset) {
 
     @Override
     public void onTransactionClick(TransactionInfo txInfo) {
-        Utils.openTransaction(txInfo, mJsonRpcService, this);
+        Utils.openTransaction(txInfo, mJsonRpcService, this, mName);
+    }
+
+    @Override
+    public void OnTxApprovedRejected(boolean approved, String accountName, String txId) {
+        pendingTxHelper.removeTransactionFromPending(accountName, txId);
```

Comment:
I don't understand that place. We are trying to get all transactions in a HashMap inside 
```
  public HashMap<String, TransactionInfo[]> getTransactions() {
        return mTxInfos;
    }
```
but they are filled only after a call `fetchTransactions` which I don't see ever called. You call only `PendingTxHelper` constructor in `initState`. `mTxInfos` is always empty I think.

Reply:
Yes, we don't need it, I got it from PortfolioFragment but missed to see the `fetchTransactions`, let me remove this and the same above places.